.github/release.yml

```diff
@@ -10,3 +10,5 @@ changelog:
       labels: ['documentation']
     - title: 'Dependency Upgrades'
       labels: ['dependencies']
+    - title: 'Internal Changes'
+      labels: ['internal', 'github_action']
```

Comment:
Do we want to convert this to a catch all category, using `*` instead of predefined labels? See example https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#example-configurations.

Reply:
Ah nice point! Should I also rename the `Internal Changes` to `Other Changes` just to be general? Or should I keep it as `Internal Changes`? I think both are good lol